doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -389,6 +389,8 @@ Matter samples
 * Updated all Matter samples that support low-power mode to enable the :ref:`lib_ram_pwrdn` feature.
   It is enabled by default for the release configuration of the :ref:`matter_lock_sample`, :ref:`matter_light_switch_sample`, :ref:`matter_smoke_co_alarm_sample`, and :ref:`matter_window_covering_sample` samples.
 
+* Updated pm static configuration for the :ref:`zephyr:nrf54l15dk_nrf54l15` target, increasing MCUboot size to enable ED25519 signature type.
+
```

Comment:
```suggestion
* Updated:

  * All Matter samples that support low-power mode to enable the :ref:`lib_ram_pwrdn` feature.
    It is enabled by default for the release configuration of the :ref:`matter_lock_sample`, :ref:`matter_light_switch_sample`, :ref:`matter_smoke_co_alarm_sample`, and :ref:`matter_window_covering_sample` samples.

  * The ``pm_static`` configuration for the :ref:`zephyr:nrf54l15dk_nrf54l15` board target, increasing MCUboot size to enable the ED25519 signature type.

```

Reply:
reopened here: https://github.com/de-nordic/fw-nrfconnect-nrf/pull/68 and applied comments resolution